packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -2571,6 +2571,51 @@ describe("calcite-combobox", () => {
           </calcite-combobox>`,
           "item3",
         ));
+
+      it("shows the selected item when initially opened", async () => {
+        const page = await newE2EPage();
+
+        await page.setContent(`
```

Comment:
Can you follow this up by:

* using the `html` formatting helper ([example](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/combobox/combobox.e2e.ts#L203))
* Removing any attributes not needed for the test – `id` doesn't seem to be used and optionally `placeholder` since the test name and assertion convey this
* adding a test for the multiple selection case (one should suffice) for extra coverage
* renaming `item1` to something that matches the test

?